ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -1113,6 +1113,49 @@ def test_paddle_tensor_bitwise_xor(
         on_device=on_device,
     )
 
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float16", "float32", "float64", "int8", "int16", "int32", "int64")},
+        "paddle",
+    )
+    def broadcast_tensors(self, name=None):
+        return paddle_frontend.Tensor(ivy.broadcast_arrays(self._ivy_array))
```

Comment:
This block of code was not supposed to be added to this file. Can you remove this?